Extensions/TileMap/helper/src/tiled/TiledTileMapLoader.spec.ts

```diff
@@ -1,9 +1,9 @@
-import { EditableTileMap, EditableTileMapLayer } from "../model/TileMapModel";
-import { TiledMap } from "./TiledFormat";
-import { TiledTileMapLoader } from "./TiledTileMapLoader";
+import { EditableTileMap, EditableTileMapLayer } from '../model/TileMapModel';
+import { TiledMap } from './TiledFormat';
+import { TiledTileMapLoader } from './TiledTileMapLoader';
 
-describe("TiledTileMapLoader", function () {
-  describe("without a collision mask", function () {
+describe('TiledTileMapLoader', function () {
+  describe('without a collision mask', function () {
```

Comment:
This file is a `spec.ts` is it ran by Karma?
I don't mind if it's converted from TypeScript to JavaScript if it's easier this way.

Reply:
I've seen tests getting run by karma for tile maps and didn't think too much beyond that but I'll check to be sure

Reply:
There are tests on the extension and other tests on the lib.

Reply:
Indeed they were not running, but they are now :+1: